FordLabs/PeopleMover
# Add metrics for creation of products and assignments

## Issue
Part of #471 - Determine Metrics Strategy

## What was done
- [x] Added product creation to tracked events
- [x] Added product assignments to tracked events
- [x] Added helper to aid in creating more events in the future

## How to test
1. Create a product
2. Assign somebody to the product
3. See that those actions wind up in [Matomo](https://matomo-fordlabs.apps.pd01i.edc1.cf.ford.com/)

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
